### PR TITLE
fix: fix brokens links

### PR DESCRIPTION
**Pull request by notdryft**

### CODE REVIEW

File: content/tutorials/getting-started/index.md [hunk 1]
```diff
@@ -167,8 +167,6 @@ If you already know about Gatling and/or you are using a Gradle, Maven or SBT pr
 * [Configuring a SBT project]({{< ref "../../reference/user/package_gen/#sbt-project" >}})
 
 Or, you can download preconfigured sample projects from Gatling Enterprise by clicking on the "Sample simulations" button on the top right in the Simulations page.
-
-Now, you can directly skip to the [generate a package]({{< ref "#generate-package" >}}) section.
```
Comment (notdryft) [inline content/tutorials/getting-started/index.md:171]: Note: this section was deleted (and was two lines below the alert?)

**Comment by slandelle** [comment]
How did you detect them?

**Comment by notdryft** [comment]
> How did you detect them?

https://github.com/gatling/gatling.io-doc-theme/pull/28

**Comment by slandelle** [comment]
I regret asking 😂

**Review by slandelle — APPROVED** [review]

**Pull request merged**